src/allmydata/scripts/tahoe_daemonize.py

```diff
@@ -166,102 +174,9 @@ def start(config):
     else:
         verb = "starting"
 
+    runner = twistd._SomeApplicationRunner(twistd_config)
```

Comment:
Ah, sad about most of the bits being private in Twisted.  I suspect this in particular may cause some trouble sooner rather than later (to the extent anything in Twisted happens "sooner") due to the twistd / twist situation and the apparent desire to move things over to twist.  OTOH, perhaps that actually means all of the twistd implementation will remain the same until it is deprecated and removed, which wouldn't be so bad for this...

codecov also says this code is uncovered, though.  If that's really the case, adding some test coverage here seems particularly important.

On the gripping hand, all `runApp` did was make a runner and then call run on it... Which is all this code appears to be doing now, albeit with an intervening `print`.  Can the print just happen before a `runApp` call and avoid the need to touch this private API?